### PR TITLE
Install base requirements for RTD

### PR DESCRIPTION
**Pull request by white238**
ReadTheDocs decided to not install a lot of packages:

https://blog.readthedocs.com/defaulting-latest-build-tools/

Built docs here:

https://axom.readthedocs.io/en/task-white238-fix_rtd/

### CODE REVIEW

**Comment by white238** [comment]
I've somehow broken the style of RTD completely.... two steps forward one back i guess.

File: .readthedocs.yml [hunk 1]
```diff
@@ -10,6 +10,10 @@ build:
   os: ubuntu-22.04
   tools:
     python: "3.11"
+  apt_packages:
+    - ghostscript
+    - graphviz
```
Comment (kennyweiss) [inline .readthedocs.yml:15]: Thanks for adding this!

Turns out that I need to merge my PR to get a fix in :)
https://github.com/LLNL/axom/pull/1171/commits/a46480514e4ec886a94351bda1b27813cbd500f6

**Review by kennyweiss — APPROVED** [review]
Thanks @white238

**Comment by white238** [comment]
I also added the logo to the top right corner. I wish we had some alternative version of the logo. for example, tranparent and a dark one.

**Comment by white238** [comment]
Fixes #1109

**Review by agcapps — APPROVED** [review]
Thank you Chris!!

**Review by rhornung67 — APPROVED** [review]
LGTM! Thanks @white238

**Pull request merged**